### DISCUSSION TITLE
How to set the transparency of the first layer(psd file)

### DISCUSSION THREAD

**Discussion by WaterYep** [ImageMagick/ImageMagick#5130] [Help]
![image](https://user-images.githubusercontent.com/53339678/167240285-00419702-297a-4254-ba24-1493935a7e06.png)

How to set the transparency of the first layer B0013.psd[0],I want to make the first layer invisible.
Thanks!

**Comment by snibgo** [comment]
What version of IM, on what platform? I assume IM v7 on Windows CMD or BAT.

The following reads all the images from a file, makes the first fully transparent, and writes the results to another file.
```
magick in.psd ( -clone 0 -alpha transparent ) -swap 0,-1  +delete out.psd
```

**Reply by WaterYep** [reply]
> What version of IM, on what platform? I assume IM v7 on Windows CMD or BAT.
> 
> The following reads all the images from a file, makes the first fully transparent, and writes the results to another file.
> 
> ```
> magick in.psd ( -clone 0 -alpha transparent ) -swap 0,-1  +delete out.psd
> ```

Thanks! It works on windows BAT, Version is ImageMagick 7.1.0-32
